### ISSUE TITLE
Enum Class: Example Code for iteration should be changed

### ISSUE THREAD

**Issue by b3bb** [dotnet/docs#5308]
see: [Iterating enumeration members ](https://docs.microsoft.com/en-us/dotnet/api/system.enum?view=netframework-4.7.2#iterating) at docs.microsoft.com

In Example Code for .Net Framework Enum Class: Section "Remarks", "Iterating enumeration members", second way to enumerate, the statement:
`ArrivalStatus status = (ArrivalStatus) Enum.ToObject(typeof(ArrivalStatus), value);` 
is useless. The resulting object `status` is equal to the input object for this statement `value`.

Proposal for a changed example:
```C#
Console.WriteLine("Members of {0}:", typeof(ArrivalStatus).Name);
foreach (ArrivalStatus status in Enum.GetValues(typeof(ArrivalStatus)))
            {
            Console.WriteLine("   {0} ({0:D})", status);
            }
```
see my [Example Code on Github](https://github.com/b3bb/MicrosoftEnumDoc/tree/master/MicrosoftEnumDoc) for a complete example (Visual Studio Solution) to play around, verify, ...

In [Program.cs](https://github.com/b3bb/MicrosoftEnumDoc/blob/master/MicrosoftEnumDoc/Program.cs) `NewVersion1` is with minimal changes to existing version, `CheckEqual` is to show the equality of status and value

**Comment by b3bb** [comment]
@BillWagner: PR in [dotnet/samples](https://github.com/dotnet/samples) and source code is samples/snippets/csharp/VS_Snippets_CLR_System/system.enum.class/cs/classiterate.cs ?
Is that correct?

**Comment by BillWagner** [comment]
Yes, @b3bb that's correct. Can you tag me in a PR? I'll watch for it.

**Comment by b3bb** [comment]
@BillWagner: Created a fork & a pull request from fork. Hope this was the correct workflow.

PR: [Enum Class: Example Code for iteration #122](https://github.com/dotnet/samples/pull/122)

**Comment by BillWagner** [comment]
Fixed in dotnet/samples#122

Closing.
